src/compute-client/src/service.rs

```diff
@@ -242,6 +252,14 @@ where
                     .map(|config| Some(ComputeCommand::CreateTimely { config, epoch }))
                     .collect()
             }
+            ComputeCommand::UpdateConfiguration(config) => {
+                self.max_result_size = config.max_result_size;
```

Comment:
I think this would have fit better into `observe_command`, which is responsible for applying consequences from commands to the state. Then the code in this method would stay responsible for splitting commands only.